packages/block-library/src/post-author-name/block.json

```diff
@@ -28,7 +28,11 @@
 		},
 		"color": {
 			"gradients": true,
-			"link": true
+			"link": true,
+			"__experimentalDefaultControls": {
```

Comment:
This block has a "Link to author archive" setting. Should we add `link` as one of the default color options?